docs/stdlib/generic.rst

```diff
@@ -44,10 +44,16 @@ Generic
 
 -----------
 
+.. note::
```

Comment:
I think using an admonition could be best to avoid duplicated documentation in unneeded places, see this for inspiration on `datetime.rst` @raddevon